code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -211,9 +223,15 @@
 /obj/machinery/capture_the_flag/proc/instagib_mode()
 	for(var/obj/machinery/capture_the_flag/CTF in machines)
 		if(CTF.ctf_enabled == TRUE)
-			CTF.ctf_gear = CTF.instagib_gear
+			CTF.current_gear = CTF.instagib_gear
```

Comment:
Couldn't you do CTF.gear = initial(CTF.gear) instead of adding current_gear and normal_gear?